docs/explanation/parts.rst

```diff
@@ -0,0 +1,263 @@
+*stage* and *prime* the build products for further processing.
+
+Describing a part
+-----------------
+
+Each part contains all the required information about a specific component,
+defining pieces of information using keys and values, as in a dictionary.
+It can also be thought of as an object with named properties.
+
+Generally, each part includes information about the following:
+ * Its `source <Source_>`_
+ * Its `dependencies <Dependencies_>`_
+ * The `build process <Build process_>`_
+ * How `build products <Build products_>`_ are exported/installed
+
+Each of these are described in the following sections.
+
+In addition, if more than one part is specified, the order in which the parts
+are built is also partially defined by its description. See `Defining the build order`_ for more information.
+
+.. ### Link to a schema or complete overview in the reference section.
+
+Source
+~~~~~~
+
+The source for a part is described using the ``source`` key. This
+specifies a location where the source code or other information is to be
+*pulled* from, and may be a repository on a remote server, a directory on
+the build host, or some other location.
+
+Additional keys are used to fine-tune the specification so that a precise
+description of the source location can be given, and also to specify the type
+of source to be processed.
+
+Where the type of the source information cannot be automatically determined,
+the ``source-type`` key is used to explicitly specify the source format.
+This influences the way in which the source code or data is processed.
+A list of supported formats can be found in the `craft_parts/sources/sources.py` file. Examples include "bzr", "git", "hg", "mercurial", "subversion", "svn", "tar", "zip", "deb", "rpm", "7z", "local".
+
+If the source type represents a file, the ``source-checksum`` key can be used
+to provide a checksum value to be compared against the checksum of the
+downloaded file.
+
+Parts with source types that describe repositories can also use additional
+keys to accurately specify where source code is found. The ``source-branch``,
+``source-commit`` and ``source-tag`` keys allow sources to be obtained from
+a specific branch, commit or tag.
+
+Since some repositories can contain large amounts of data, the
+``source-depth`` key can be used to specify the number of commits in a
+repository's history that should be fetched instead of the complete history.
+For repositories that use submodules, the ``source-submodules`` key can be
+used to selectively fetch only those submodules that are needed.
+
+The ``source-subdir`` key specifies the subdirectory in the unpacked sources
+where builds will occur. **Note:** This key restricts the build to the
+subdirectory specified, preventing access to files in the parent directory
+and elsewhere in the file system directory structure.
+
+Dependencies
+~~~~~~~~~~~~
+
+The dependencies of a part are described using the ``build-snaps`` and
```

Comment:
Maybe "build-time dependencies", as opposed to runtime dependencies? Or even "Components required during the part build process"?